docs/pages/reference/helm-reference/includes/zz_generated.teleport-kube-agent.mdx

```diff
@@ -0,0 +1,1627 @@
+
+## `roles`
+
+| Type | Default |
+|------|---------|
+| `string` | `"kube"` |
+
+`roles` is a comma-separated list of services which should be enabled
+when running the `teleport-kube-agent` chart.
+
+| Services                     | Value for `roles` | Mandatory additional settings for this role |
+|------------------------------|-------------------|---------------------------------------------|
+| Teleport Kubernetes service  | `kube`            | [`kubeClusterName`](#kubeclustername)       |
+| Teleport Application service | `app`             | [`apps`](#apps) or [`appResources`](#appresources) |
+| Teleport Database service    | `db`              | [`databases`](#databases) or [`databaseResources`](#databaseresources) |
+| Teleport Discovery service   | `discovery`       | [`kubeClusterName`](#kubeclustername) |
+
+For example: `roles: kube,app,discovery`.
+
+## `proxyAddr`
+
+| Type | Default |
+|------|---------|
+| `string` | `""` |
+
+`proxyAddr` provides the public-facing Teleport proxy endpoint which should be used to join the cluster. This is the same URL that is used
+to access the web UI of your Teleport cluster. It is the same as the value configured for `proxy_service.public_addr` in a traditional
+Teleport cluster. The port used is usually either 3080 or 443.
+
+Here are a few examples:
+
+| Deployment method             | Example `proxy_service.public_addr` |
+|-------------------------------|-------------------------------------|
+| On-prem Teleport cluster      | `teleport.example.com:3080`         |
+| Teleport Cloud cluster        | `example.teleport.sh:443`           |
+| `teleport-cluster` Helm chart | `teleport.example.com:443`          |
```

Comment:
I'm not sure we need this table. If users aren't sure which port the Proxy Service listens on, maybe we should add a `curl <proxy_addr>/webapi/ping` example?